### PR TITLE
Yanchi Liu/HW1

### PR DESCRIPTION
**Pull request by Yanchi-Liu**
It's a relatively easy work to do, but I still learn a lot, especially the details, from doing it. The most difficult part in terms of my experiences is the linkage process between GitHub and R, since this is the first time I use GitHub as a cloud-share system. All in all, I enjoy the class and the homework as well.

### CODE REVIEW

**Comment by wanitchayap** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |     X      |              |            |

Remarks:

* It is a great practice to commit regularly like you did!
* You completed all the requirements for the HW. Great job :)
* I'm glad you enjoy the class and the HW.

**Review by wanitchayap — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |          |       X       |            |

Remarks:

* It is a great practice to commit regularly like you did!
* You completed all the requirements for the HW. Great job :)
* I'm glad you enjoy the class and the HW.
* Make sure you commit files through RStudio, not uploading through github. This will result in the reduction of point in the ease-of-access score in subsequent assignments.

File: README.md [hunk 1]
```diff
@@ -1 +1,25 @@
-# hw01
\ No newline at end of file
+# A Profile of Yanchi Liu
+## Introduction
+Hi, everyone. This is Yanchi Liu, a current student in MAPSS Econ. I'm from Chengdu, China, a city famous for the hot-pot and pandas (see this [link](https://en.wikipedia.org/wiki/Chengdu) to find more attractive information about Chengdu). This is my first time visiting and living in US and I'm eager to explore the academic life here, as well as the city of Chicago.
```
Comment (wanitchayap) [inline README.md:3]: Great link embedding!

File: README.md [hunk 2]
```diff
@@ -1 +1,25 @@
...
+B.A., Economics, Nankai University, 2019.
+
+## Research Interests
+My research interest generally focuses on **International Trade**
+
+1. Measurement and Changes of Export Product Qualities under Trade Liberalization
+2. Import Structures and the Influences on Domestic Labor Market
+
+## Hobbies
+* *Travelling*
```
Comment (wanitchayap) [inline README.md:17]: Nice combination between list and *italic*!

**Comment by mengdihuang** [comment]
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    x       |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |      x     |              |            |
| **Ease of access**          |     x      |              |            |

The bio is really clear since you headered all of your topics!
And I also love that you added a link next to Chengdu! Makes it very easy to get to know your city!
Overall great job!

File: README.md (hunk 1 above)
Comment (mtlynch4) [inline README.md:3]: Good use of linking for those who do not know Chengdu

File: README.md [hunk 3]
```diff
@@ -1 +1,25 @@
...
+# A Profile of Yanchi Liu
+## Introduction
+Hi, everyone. This is Yanchi Liu, a current student in MAPSS Econ. I'm from Chengdu, China, a city famous for the hot-pot and pandas (see this [link](https://en.wikipedia.org/wiki/Chengdu) to find more attractive information about Chengdu). This is my first time visiting and living in US and I'm eager to explore the academic life here, as well as the city of Chicago.
+
+## Education
+M.A. Candidate, Economics, The University of Chicago, 2019-present.
+
+B.A., Economics, Nankai University, 2019.
+
+## Research Interests
```
Comment (mtlynch4) [inline README.md:10]: Frequently separating the document into different sections with their own headers makes it easy to read.

File: README.md (hunk 2 above)
Comment (mtlynch4) [inline README.md:17]: I said this on my last review as well, but I didn't realize you could make lists this way! Helpful to know

File: README.md [hunk 4]
```diff
@@ -1 +1,25 @@
...
+1. Measurement and Changes of Export Product Qualities under Trade Liberalization
+2. Import Structures and the Influences on Domestic Labor Market
+
+## Hobbies
+* *Travelling*
+* *Pop Music*
+* *Cycling*
+
+## My Photo
+![](Yanchi.png)
```
Comment (mtlynch4) [inline README.md:22]: I saw when I ran this in my R that you included the photo with the code, making it easier for other users to see the photo. It made me realize that I could have done better when I linked my photo to the local location, which made the code messier (I had to include the path) and made it harder for other users to see it.

**Review by mtlynch4 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |    x       |              |            |

Remarks:
-Overall, well done. The code is easy to read, runs well on my computer, and all the requirements are met. 
-I appreciated the frequent commits, and the comments explaining their purpose especially. 
-Good job!

**Pull request closed without merging**